xdem/coreg.py

```diff
@@ -280,26 +290,35 @@ def icp_coregistration(reference_dem: np.ndarray, dem_to_be_aligned: np.ndarray,
         reference_dem[mask] = np.nan
         dem_to_be_aligned[mask] = np.nan
 
+    # Generate center point coordinates for each pixel
```

Comment:
You know there's a function that does just this in Raster? self.coords

Reply:
I did not know that! Good to know!

Right now, the coreg methods work completely independently of the Raster class (Rasters are only used in the main `coregister()` function).

This could be changed in the future, but I propose that it should be a separate "simplification PR"

Reply:
I agree that georeferenced-based functions such as coords() should be kept to a minimum :+1: